media/js/firefox/concerts.js

```diff
@@ -13,7 +13,7 @@ function onYouTubeIframeAPIReady() {
 }
 
 
-(function(Mozilla) {
+(function (Mozilla) {
```

Comment:
Are these stylistic changes, or are you having issues with your editor's ESLint config? In terms of bedrock, we don't have any rules set about whitespace changes like this (afaik)?

Reply:
These changes were made when I selected `Format Document` in vscode. Seems to be the default formatting.
I added `"javascript.format.insertSpaceBeforeFunctionParenthesis": false` to my `settings.json` file in my editor so hopefully that will prevent the difference in the future.
I also added `"eslint.autoFixOnSave": true` to ensure eslint is being applied on save. :)

Reply:
I'm not quite sure what kind of rules VSCode uses with the `Format Document` command, but I'd recommend to try installing [this ESLint plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). Once installed it should use the version of ESLint that's found locally in the `node_modules` folder within the workspace, and will also use the rules defined in our root `.eslintrc`. The tool will also lint as you type, which is pretty cool :)

Reply:
Totally! I have the ESLint plugin, I just made the mistake of using `Format Document` which overrode with it's own formatting opinions. 😅 I couldn't find a way to correct with ESlint because our `.eslintrc` file doesn't appear to have opinions about spacing.

I found that if I added to `settings.json` both: 
`"javascript.format.insertSpaceBeforeFunctionParenthesis": false` and `"javascript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false` it reverted back to our preferred spacing format.